src/frontend/komponenter/Brev/stønadsverdier/preutfylleVedtaksDatoer.ts

```diff
@@ -1,7 +1,7 @@
 import { Brevverdier } from './verdier';
 import { formaterNullableTilTekstligDato } from '../../../utils/dato';
 
-export const lagVerdierVedtakFraOgTil = (
+export const preutfylleVedtaksDatoer = (
```

Comment:
Denne prefyller ju egentlige ikke, den mapper verdier som kan brukes for å prefylles 👀